templates/cisco_wlc_ssh_show_time.textfsm

```diff
@@ -0,0 +1,30 @@
+Value TIME (.+)
+Value TZ_DELTA (\S+)
+Value TIMEZONE (.+)
+Value TZ_NAME (.+)
+Value NTP_VERSION (\d)
+Value NTP_POLLING_INTERVAL (\d+)
+Value INDEX (\d+)
+Value NTP_KEY_INDEX (\d+)
+Value NTP_SERVER (\S+)
+Value STATUS (\S+\s?\S+)
+Value NTP_MSG_AUTH_STATUS (\S+\s?\S+)
+
+Start
+  ^\s*Time\s*\.+\s+${TIME}\s*$$
+  ^\s*Timezone [dD]elta\s*\.+\s+${TZ_DELTA}\s*$$
+  ^\s*Timezone [lL]ocation\s*\.+\s+\(${TIMEZONE}\)\s+${TZ_NAME}\s*$$
+  ^\s*NTP [sS]ervers\s*$$
+  ^\s*NTP [vV]ersion\s*\.+\s+${NTP_VERSION}\s*$$
+  ^\s*NTP [pP]olling [iI]nterval\s*\.+\s+${NTP_POLLING_INTERVAL}\s*$$ -> Record
```

Comment:
With the comment above, I'd remove this `Record` and just let it record at the NTP servers so it'll parse two entries with the same top level information.

Reply:
@FragmentedPacket 
I removed the Record so the entries/parsed data is more succinct. Thank you!
Oddly enough there's an extra line being picked up, but I can't seem to figure out what's adding it.

Reply:
Ah ok. Make the `index` required and I think that should fix the issue.

Reply:
> Ah ok. Make the `index` required and I think that should fix the issue.

That did it. Thank you!
Fixed up and changes have been pushed to my branch.